trino/client.py

```diff
@@ -839,3 +768,147 @@ def decorated(*args, **kwargs):
         return decorated
 
     return wrapper
+
+
+class RowMapperFactory:
+    """
+    Given the 'columns' result from Trino, generate a list of
+    lambda functions (one for each column) which will process a data value
+    and returns a RowMapper instance which will process rows of data
+    """
+
+    def create(self, columns, experimental_python_types):
+        assert columns is not None
+
+        if experimental_python_types:
+            return RowMapper([self._col_func(column['typeSignature']) for column in columns])
+        return RowMapper()
```

Comment:
Create a well-named singleton for the `NoOpRowMapper`/`PassThroughRowMapper`.

Reply:
Done